public/app/percona/dbaas/components/DBCluster/EditDBClusterPage/DBClusterAdvancedOptions/DBClusterAdvancedOptions.constants.ts

```diff
@@ -34,16 +29,16 @@ export const DEFAULT_SIZES: DBClusterDefaultResources = {
 };
 
 export const INITIAL_VALUES = {
-  topology: DBClusterTopology.cluster,
   nodes: 3,
   single: 1,
   resources: DBClusterResources.small,
   memory: DEFAULT_SIZES.small.memory,
   cpu: DEFAULT_SIZES.small.cpu,
   disk: DEFAULT_SIZES.small.disk,
+  sourceRanges: [{}],
```

Comment:
I've seen this in other places. Would a simple empty array crash anything?

Reply:
yep

Reply:
https://codesandbox.io/s/kx8qv67nk5 - this is an example from codesandbox, it shows how it works. It doesn't show value in initial state, so we need to add some object to array for showing field.